src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1282,12 +1313,26 @@ private Task<INetworkInterface> AssociateNicWithNetworkSecurityGroupAsync(INetwo
                 () => networkInterface.Update().WithExistingNetworkSecurityGroup(networkSecurityGroup).ApplyAsync()
             );
 
-        private Task CreateMySqlDatabaseUser(ConnectionInfo sshConnectionInfo, Server mySqlServer)
+        private Task CreatePostgreSqlDatabaseUser(ConnectionInfo sshConnectionInfo)
             => Execute(
-                $"Creating MySQL database user...",
-                () => ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"mysql -u {configuration.MySqlAdministratorLogin} -p'{configuration.MySqlAdministratorPassword}' -h {mySqlServer.FullyQualifiedDomainName} -P 3306 < {CromwellAzureRootDir}/mysql-init/init-user.sql")
+                $"Creating PostgreSQL database user...",
+                () => ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"psql postgresql://{configuration.PostgreSqlAdministratorLogin}:{configuration.PostgreSqlAdministratorPassword}@{configuration.PostgreSqlServerName}.postgres.database.azure.com/{configuration.PostgreSqlDatabaseName} < {CromwellAzureRootDir}/mysql-init/init-user-postgre.sql")
             );
 
+        private Task<IPrivateDnsZone> CreatePrivateDnsZoneAsync(INetwork virtualNetwork)
+            => Execute(
+                "Creating private DNS Zone for PostgreSQL Server",
+                async () =>
+                {
+                    // Note: upon refactoring out of fluent see commit cbffa28 in #392. 
```

Comment:
Unclear what this comment means; would recommend deleting or clarifying it

Reply:
Got it... maybe something like this might help clarify?

```
// Note: for a potential future implementation of this method without Fluent,
// please see commit cbffa28 in #392
```

Reply:
updated- thanks for the clarification :)